cmake/build_configurations/mysql_release.cmake

```diff
@@ -272,4 +277,20 @@ IF(UNIX)
       ENDIF()
     ENDIF()
   ENDIF()
+
+  # AIX flags
```

Comment:
mostly curiosity,  but do you need that?
does  not normal build  (Which defaults to -DCMAKE_BUILD_TYPE=RelWithDebInfo) work alright?
That -O3 was used for compatibility with Linux autotools build, 10 years ago (because some people insisted that not a single compile option change when switching build systems), and remained since.

Now , it may not be best idea. -O2 , the RelWithDebInfo flag, should be reasonably good.

Reply:
I use the same flags than default GCC, so ``-g -static-libgcc -fno-omit-frame-pointer -fno-strict-aliasing  -Wno-uninitialized``, except `-static-libgcc` is not working on AIX.

About `-O3`, it is consistent with all other platforms.

About RELEASE and not RELWITHDEBINFO, I think it is an old stuff. I update it.

Reply:
Well, "default" compilation does not use any of these flags.
default compilation is

cmake . 
make

it did not build well?  why it is necessary to add more flags?

Reply:
I have tried, and CMake checks and adds automatically these flags. So, build will be exactly the same, and these lines are useless.